src/IK/include/BiomechanicalAnalysis/IK/InverseKinematics.h

```diff
@@ -402,7 +402,7 @@ class HumanIK
      * @param jointVelocities joint velocities, by defualt it is set to zero
      * @return true if the joint regularization task is set correctly
      */
-    bool updateJointRegularizationTask();
+    bool updateJointRegularizationTask(const Eigen::VectorXd& desiredPositionsKp, const std::vector<std::string> jointsListKp);
```

Comment:
This is a breaking change and would break the iFeelBAF internal application, did you coordinated for doing a change there?

Reply:
No problem, I just saw the internal PR. Anyhow, I would clearly document this change in the PR body, for example with this text:

> This PR changes the signature of `HumanIK::updateJointRegularizationTask()` to be `HumanIK::updateJointRegularizationTask(const Eigen::VectorXd& desiredPositionsKp, const std::vector<std::string> jointsListKp)` . If you want to continue to not regularize any joint, change your code from:
> 
> ~~~cpp
> humanIK.updateJointRegularizationTask();
> ~~~
>
> to
> 
> ~~~cpp
> Eigen::VectorXd dummyVec;
> std::vector<std::string> dummyList;
> humanIK.updateJointRegularizationTask(dummyVec, dummyList);
> ~~~